src/openrct2/ride/Vehicle.h

```diff
@@ -402,8 +402,7 @@ enum {
     VEHICLE_SOUND2_FLAGS_LIFT_HILL = 1 << 1 // When on a lift hill generate sound
 };
 
-enum friction_sound
-{
+enum {
```

Comment:
Why did you change the brace placement? According to our code style, braces should be on their own lines.

Reply:
it was based on other nameless enums in vehicle.h